src/main/kotlin/org/jetbrains/intellij/platform/gradle/models/IvyModule.kt

```diff
@@ -71,7 +75,16 @@ data class IvyModule(
  * <ivy-module version="2.0">
  *     ...
  *     <publications>
- *         <artifact conf="default" ext="jar" name="/path/to/artifact.jar" type="jar" />
```

Comment:
Previously in all paths we had the leading slash, but the artifact pattern was already `/[artifact]` which would lead to double slash eventually. It seems to work like that, but I fixed this anyway.